packages/devtools_app/lib/src/framework/framework_core.dart

```diff
@@ -26,6 +27,7 @@ class FrameworkCore {
     setGlobal(MessageBus, MessageBus());
     setGlobal(FrameworkController, FrameworkController());
     setGlobal(SurveyService, SurveyService());
+    setGlobal(ProgramExplorerController, ProgramExplorerController());
```

Comment:
why is this a global instead of being part of the debugger controller?

Reply:
Mostly so it can be mocked out in tests. I originally had it as a member of the debugger controller but ran into issues when we were testing the non-mock DebuggerController.